### ISSUE TITLE
Return of offline session bug on 26.0.0

### ISSUE THREAD

**Issue by mixman68** [keycloak/keycloak#33610]
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

core

### Describe the bug

I get the same error of #32832 or #32127 on KeyCloak 26.0.0 after an upgrade from 25.0.1.

Database provider is postgresql

### Version

26.0.0

### Regression

- [X] The issue is a regression

### Expected behavior

User can login

### Actual behavior

User get a error 500 and this error is logged :

```
2024-10-05 12:49:19,334 INFO  [org.hibernate.event.internal.DefaultLoadEventListener] (org.keycloak.models.sessions.infinispan.changes.PersistentSessionsWorker$BatchWorker) HHH000327: Error performing load command: org.hibernate.exception.SQLGrammarException: JDBC exception executing SQL [select pcse1_0.CLIENT_ID,pcse1_0.CLIENT_STORAGE_PROVIDER,pcse1_0.EXTERNAL_CLIENT_ID,pcse1_0.OFFLINE_FLAG,pcse1_0.USER_SESSION_ID,pcse1_0.DATA,pcse1_0.TIMESTAMP,pcse1_0.VERSION from public.OFFLINE_CLIENT_SESSION pcse1_0 where (pcse1_0.CLIENT_ID,pcse1_0.CLIENT_STORAGE_PROVIDER,pcse1_0.EXTERNAL_CLIENT_ID,pcse1_0.OFFLINE_FLAG,pcse1_0.USER_SESSION_ID) in ((?,?,?,?,?))] [ERROR: column pcse1_0.version does not exist
```

### How to Reproduce?

Take a keycloak 25.0.1, upgrade to 26.0.0 and try to login

**Comment by ahus1** [comment]
I'm sorry to hear this upgrade is causing troubles for you. The other bugs you referenced were related to MSSQL. As you are using PostgreSQL, this one seems to be different. 

Can you please check your database and have a look at the table structure of `OFFLINE_CLIENT_SESSION` and `OFFLINE_USER_SESSION`? They should have a column `VERSION`, which should have been added in Release 25.0.0 already. If it doesn't exist, maybe the database schema migration wasn't performed?

**Comment by mixman68** [comment]
Hi
Thanks for y reply

Column version is missing, can you give me the alter query to add this please ?

is it cuz I jumped from 22 to 25.0.1 ?

**Comment by ahus1** [comment]
By default, Keycloak will catch up with any database migrations automatically, even if you skip versions. 

You can opt-out of automatic migrations, see "migration strategy" for the details: https://www.keycloak.org/server/db#_setting_jpa_provider_configuration_option_for_migrationstrategy
Please review your Keycloak configuration if this is the case for you that you opted out of the migration. You can also share you configuration here for review (it might be stored in a file, in CLI options, or environment variables). 

All migrations that have run on your database are recorded in the table `DATABASECHANGELOG`. For this specific change, it should have been recorded with an entry of `id` set to `25.0.0-28265-tables`.

**Comment by mixman68** [comment]
Hi

Seems to be solved, the database migration is here 25.0.0-28265-tables but without column

so I started an empty database, dump structure of new database, ans data from mine and restore structure of new database into mine and data of mine and all is ok

**Comment by ahus1** [comment]
Thank you for the update. It still worries me that the column is missing. I hope this will fix your issues.
